### ISSUE TITLE
Choosing About RStudio multiple times displays a separate dialog for each click

### ISSUE THREAD

**Issue by ronblum** [rstudio/rstudio#2481]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.462
    OS Version      : 10.12.6 (Sierra)
    R Version       : 3.4.4

### Steps to reproduce the problem

1. `Help -> About RStudio`
2. `Help -> About RStudio`
3. `Help -> About RStudio`
 
### Describe the problem in detail

The `About RStudio` dialog appears three times. This is easier to see if they're moved around, but also closing the top one by clicking `OK` will reveal the second one behind it, and the same with clicking `OK` on that one.

### Describe the behavior you expected

Only one About RStudio dialog should be displayed. At most, if it's moved around after the first time, choosing About RStudio a second time should cause the dialog to recenter, although that's not necessary.

**Comment by ronblum** [comment]
Note: this isn't the only dialog like that. Other dialogs, like `Tools -> Install Packages` and `View -> Panes -> Pane Layout` behave the same way.

**Comment by ronblum** [comment]
Maybe same or related to #2482

**Comment by ronblum** [comment]
This is no longer happening with the About dialog, but it's still happening with the others.
